packages/arb-token-bridge-ui/src/components/TransferPanel/DepositConfirmationDialog.tsx

```diff
@@ -18,7 +18,7 @@ import { BridgesTable } from '../common/BridgesTable'
 import { useAppState } from '../../state'
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
 import { getNetworkName, isNetwork } from '../../util/networks'
-import { ReactComponent as CustomClipboardCopyIcon } from '../../assets/copy.svg'
+import CustomClipboardCopyIcon from '../../assets/copy.svg'
```

Comment:
why do we change the other ones to `<img/>` but keep this import?

Reply:
@fionnachan the others were imported as a default import, which gives you their URL that you can use as `src`, while this was an SVG imported as a React component.

Reply:
For the other comment, :100: agree. Created an issue here: https://github.com/OffchainLabs/arb-token-bridge/issues/558